src/test/resources/com/atomist/rug/test/gherkin/handler/event/PassingFeature2Steps.ts

```diff
@@ -0,0 +1,14 @@
+import {Given,When,Then, HandlerScenarioWorld} from "@atomist/rug/test/handler/Core"
+import * as node from "../../../handlers/event/Nodes"
+
+When("i call you", world => {
+   world.registerHandler("ReturnsAMessage")
+   world.sendEvent(new node.Commit)
+})
+Then("you call me", world => {
+    return world.message() != null
```

Comment:
Well, the TS interface says you return a Plan or a Message, so there is an inconsistency between Scala and TypeScript. As a user, I certainly know about the latter not the former. I think, while testing a handler returning a message, I expected to ask for the message from `world`.

With that said, I do like the idea of always returning a plan but this is not something that is obvious from the TS side I'd say.